src/api/auth.api.ts

```diff
@@ -1,55 +1,63 @@
-import axios from "axios";
+import axios from 'axios'
 
-const BASE_URL = process.env.REACT_APP_BASE_URL || "api.comidit.app"
+export const BASE_URL = process.env.REACT_APP_BASE_URL || 'api.comidit.app'
 
-export type User = Omit<BackUser, "address"> & Address;
+export type User = Omit<BackUser, 'address'> & Address
 
-interface BackUser {
-	"role"           :string,
-	"name"           :string,
-	"email"          :string,
-	"password"       :string,
-	"phone"          :number,
-  "address"        :Address[],
+export interface BackUser {
+  role: string
+  name: string
+  email: string
+  password: string
+  phone: number
+  address: Address[]
 }
 
 const toBackUser = (user: User): BackUser => {
   return {
-	"role"            :user.role,
-	"name"            :user.name,
-	"email"           :user.email,
-	"password"        :user.password,
-	"phone"           :+user.phone,
-    "address": [{
-      "street"      :user.street,
-      "streetNumber":user.streetNumber,
-      "country"     :user.country,
-      "zipCode"  :+user.zipCode,
-      "city"        :user.city
-    }],
+    role: user.role,
+    name: user.name,
+    email: user.email,
+    password: user.password,
+    phone: +user.phone,
+    address: [
+      {
+        street: user.street,
+        streetNumber: user.streetNumber,
+        country: user.country,
+        zipCode: +user.zipCode,
+        city: user.city,
+      },
+    ],
   }
 }
 
 export interface LoginUser {
-  "email"          :string,
-	"password" :string,
+  email: string
+  password: string
 }
 
 interface Address {
-  "street": string
-  "streetNumber": number
-  "country": string
-  "zipCode": number
-  "city": string
+  street: string
+  streetNumber: number
+  country: string
+  zipCode: number
+  city: string
 }
 
 export const signUp = (newUser: User) => {
   console.log(toBackUser(newUser))
-  axios.post(`${BASE_URL}auth/signup`, toBackUser(newUser))
+  axios
+    .post(`${BASE_URL}auth/signup`, toBackUser(newUser))
     .then((response) => console.log(response))
 }
 
 export const signIn = (user: LoginUser) => {
-  axios.post(`${BASE_URL}auth/signin`, user)
-  .then((response) => console.log(response))
+  axios
+    .post(`${BASE_URL}auth/signin`, user)
+    .then((response) => console.log(response))
+}
+
+export const getRestaurants = () => {
+  axios.get(`${BASE_URL}restaurants`).then((response) => console.log(response))
 }
```

Comment:
Con el paso a React Query, esto ya no hace falta. Los otros tampoco pero falta pasarlos a dentro de su componente
```suggestion
export const getRestaurants = () => {
  axios.get(`${BASE_URL}restaurants`).then((response) => console.log(response))
}
```